optics/xmmTrueWolter.rml

```diff
@@ -0,0 +1,31 @@
+<xml>
+	<globals>
+		<constant name="pi" value="3.1415927" />
+	</globals>
+
+	<TRestAxionWolterOptics name="xmm" verboseLevel="warning" >
```

Comment:
Has to be changed into TRestAxionTrueWolterOptics as soon as that exists

Reply:
Then, perhaps it could be already named `TRestAxionTrueWolterOptics` even if it does not exist.

Reply:
Yeah, you're right^^